htdocs/public/webportal/tpl/footer.tpl.php

```diff
@@ -0,0 +1,102 @@
+<?php
+// Protection to avoid direct call of template
+if (empty($context) || !is_object($context)) {
+	print "Error, template page can't be called as URL";
+	exit;
+}
+
+global $langs;
+
+// load messages
+$html = '';
+$htmlSuccess = '';
+$htmlWarning = '';
+$htmlError = '';
+$jsOut = '';
+$jsSuccess = '';
+$jsWarning = '';
+$jsError = '';
+//$useJNotify = false;
```

Comment:
I see jquery is included but just for jnotify library support (and dialog box, see my previous comment)
it could be a good thing to try to set webportal jquery free 
If @eldy is interested I can see for a vanillia js notification class and css and make a code suggestion. (but I prefer skip spend time if @eldy isn't interrested)

Reply:
Post-scriptum : 

For information : if we need a multiselect  form, it's already managed by `<details role="list">`
See https://picocss.com/docs/dropdowns.html in dropdown section : select multiple elements

Reply:
Currently notification using jquery is already part of core, so we can continue to use jnotify